### PR TITLE
4-Deployment\2-deploy-static

### PR DESCRIPTION
**Pull request by diberry**
## Purpose

1. Remove unnecessary step to add Graph User.Read - it is now created by default for every app registration (I tried single and mulit-tenant).
2. Changed display of sensitive data from actual value to ellipsis. 

I'm not sure the step to change the manifest file's value of accessTokenAcceptedVersion is necessary. I didn't do it but the client and API still worked as I expected. What error or lack of functionality should I have seen from not setting it?

## Does this introduce a breaking change?

```
[ ] Yes
[x ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x ] Bugfix
[ x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: 4-Deployment/2-deploy-static/README.md [hunk 1]
```diff
@@ -78,17 +78,13 @@ The first thing that we need to do is to declare the unique [resource](https://d
...
-     - In the **Delegated permissions** section, select the **User.Read** in the list. Use the search box if necessary.
-     - Select the **Add permissions** button at the bottom.
+   - Keep the Graph API User.Read permission, already provided by default.
    - Select the **Add a permission** button and then,
      - Ensure that the **My APIs** tab is selected.
      - In the list of APIs, select the API `msal-react-spa`.
      - In the **Delegated permissions** section, select the **Access 'msal-react-spa'** in the list. Use the search box if necessary.
      - Select the **Add permissions** button at the bottom.
-1. In the app's registration screen, select the **Manifest** blade. Then:
+2. In the app's registration screen, select the **Manifest** blade. Then:
```
Comment (derisen) [inline 4-Deployment/2-deploy-static/README.md:87]:
```suggestion
2. In the app's registration screen, select the **Manifest** blade. Then:
```

Is it needed to set the step number to "2" here? Wouldn't this be automatically numbered?

**Review by derisen — APPROVED** [review]
LGTM! left one comment. 

About setting `accessTokenAcceptedVersion` to 2. The token validation logic in the API side is assuming the token version is 2 for the purpose of getting signing keys. `null` defaults to 1.

Not sure how it didn't cause a problem with in your test. Was the audience in your app registration single-tenant or multi-tenant (i.e. `tenant Id` vs `common`) by chance? Azure AD sets it to 2 by default in case the app supports sign-in with multiple tenants and personal microsoft accounts..

**Comment by diberry** [comment]
@derisen I did start out with a multi-tenant app, then switch later to a single tenant - that was the issue, thanks. Is there an #term to sign-off and merge this PR?

**Comment by derisen** [comment]
@diberry ah, that explains! 

No such commands on this repo : ( let me know when ready and I'll hit merge

**Comment by diberry** [comment]
I'm ready.

**Pull request merged**